tests/chromium/session.spec.ts

```diff
@@ -37,10 +37,10 @@ it('should send events', async function({page, server}) {
   expect(events.length).toBe(1);
 });
 
-it('should only accept a page', async function({page}) {
-  // @ts-expect-error newCDPSession expects a Page
+it('should only accept a page or frame', async function({page}) {
+  // @ts-expect-error newCDPSession expects a Page or Frame
   const error = await page.context().newCDPSession(page.context()).catch(e => e);
-  expect(error.message).toContain('page: expected Page');
+  expect(error.message).toContain('frame: expected Frame');
```

Comment:
I "fixed" this up. The error was automagically coming from channelOwner.ts since context is channelable. Looks like the error might depend on how the object props are iterated!